Sources/MarathonCore/Marathon.swift

```diff
@@ -40,8 +40,9 @@ public final class Marathon {
             let rootFolder = try fileSystem.createFolderIfNeeded(at: folderPath)
             let packageFolder = try rootFolder.createSubfolderIfNeeded(withName: "Packages")
             let scriptFolder = try rootFolder.createSubfolderIfNeeded(withName: "Scripts")
+            let autocompletionsFolder = try rootFolder.createSubfolderIfNeeded(withName: "Autocompletions")
```

Comment:
How about naming this `ShellAutocomplete` instead? That way it's clear that it's for shells.